### PR TITLE
add test for the no-parent case

### PR DESCRIPTION
**Pull request by lgdean**
#73 
I think this finishes things up, as the actual error cases are explicitly to be handled in later work. Please let me know if I've missed things, though!

### CODE REVIEW

File: parser/shared/src/test/scala/hmda/parser/util/FITestData.scala [hunk 1]
```diff
@@ -4,6 +4,8 @@ object FITestData {
 
   val tsDAT =
     "100110132019201501171330 201320-1177984  551  SMALL BANK USA, NA            1850 TYSONS BLVD., SUITE 50             MCLEAN                   VA22102     BIGS USA, INC.                412 THIRD AVENUE                        NEW YORK                 NY10012     Bob Smith                     555-555-5555999-999-9999bob.smith@bank.com                                                "
+  val tsDAT_noParent =
```
Comment (hkeeler) [inline parser/shared/src/test/scala/hmda/parser/util/FITestData.scala]: Scala variable names should be camelCase.

[Scala Style Guide](http://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods) says:
>Method, Value and variable names should be in lower camel case

**Comment by hkeeler** [comment]
Overall, this looks good, just the one nitpick about the variable name.

File: parser/shared/src/test/scala/hmda/parser/util/FITestData.scala (hunk 1 above)
Comment (lgdean) [inline parser/shared/src/test/scala/hmda/parser/util/FITestData.scala]: I see your point. So, tsDATNoParent or tsDATnoParent? The former seems a little hard to skim, but the latter is inconsistent. (So I suppose I'd lean toward the former.)

Comment (hkeeler) [inline parser/shared/src/test/scala/hmda/parser/util/FITestData.scala]: Agreed.  I think `tsDATNoParent` is better...or `tsDatNoParent`, but then you have to change the other variable name to make it consistent. :)  Either is fine by me.

**Comment by hkeeler** [comment]
Looks good.  Merging...

Closes #73

**Pull request merged**